modular_skyrat/master_files/code/datums/traits/negative.dm

```diff
@@ -65,3 +65,27 @@
 	value = -4
 	mob_trait = TRAIT_NOGUNS
 	icon = "none"
+
+/datum/quirk/sensitivesnout
+	name = "Sensitive Snout"
+	desc = "Your snout has always been sensitive, and it really hurts when someone pokes it!"
+	gain_text = span_notice("Your snout is awfully sensitive.")
+	lose_text = span_notice("Your snout feels numb.")
+	medical_record_text = "Patient's snout seems to have a cluster of nerves in the tip, would advise against direct contact."
+	value = -2
+	mob_trait = TRAIT_SENSITIVESNOUT
+	icon = "fingerprint"
+
+/datum/quirk/sensitivesnout/post_add()
+	quirk_holder.apply_status_effect(/datum/status_effect/sensitivesnout)
```

Comment:
make this an element instead

Reply:
which part? the status effect or the post add? I am confused by what you mean, as having it DCS is kinda complicated for a single use single event?

Reply:
the entire status effect should be an element instead, and it should add the text to the mob when the examine signal fires

Reply:
oh it is done this way because of how the rest of the code I observed that added these messages worked